index.html

```diff
@@ -3885,6 +3885,29 @@ <h3>JSON-LD</h3>
 base media type, `application/vc+ld+json`.
         </p>
 
+        <p>
+As elaborated upon in Section <a href="#static-processing"></a>, some software
+applications might not perform dynamic JSON-LD processing, which uses features
+such as JSON-LD compaction and expansion. Static processors can elect to reject
+unrecognized URL values in the `@context` property, for example. Authors of
+<a>conforming documents</a> are also advised that use of certain dynamic
+JSON-LD processor features might reduce interoperability. These features
+include the following:
+        </p>
+
+        <ul>
+          <li>
+Use of JSON-LD keywords in the `@context` value that globally modify document
+value processing, such as global settings of `@base` and `@language`
+          </li>
+          <li>
+Use of full URLs for JSON-LD terms and types, instead of the compact forms of
+those values as are defined in application-specific JSON-LD context files
```

Comment:
```suggestion
Use of full URLs for JSON-LD terms and types (e.g.,
`https://www.w3.org/2018/credentials#VerifiableCredential`), instead of the
unmapped forms of any such values (e.g., `VerifiableCredential`) that are
among active JSON-LD `@context` mappings (e.g.,
`https://www.w3.org/ns/credentials/v2`)
```